docs/index.html

```diff
@@ -107,6 +109,25 @@ <h1>makeup-expander</h1>
                     </div>
                 </div>
             </span>
+
+            <div class="expander expander--click-only-nested flyout">
+                <span class="expander__host"><button class="flyout__host">Click for Flyout - nested button</button></span>
```

Comment:
Can we use an input instead of a button for the example?

Reply:
Yes! I completely forgot about this.